lib/connect_proto/src/address.proto

```diff
@@ -1,5 +1,6 @@
 syntax = "proto3";
 package primary.connect;
+option go_package = "primary.connect";
```

Comment:
Assuming/guessing that the Go generator doesn't pick up on the `package` directive? That's kinda lame